app/components/Invoke/index.js

```diff
@@ -0,0 +1,411 @@
+            value={selectedInvokeAddress}
+            onChange={this.onChangeInvokeAddress}
+          >
+            {addresses.map(address => (
+              <ItemWrapper
+                component="div"
+                key={address.pkh}
+                value={address.pkh}
+              >
+                <TezosAddress
+                  address={address.pkh}
+                  size="16px"
+                  color="gray3"
+                  color2="primary"
+                />
+                <SpaceBar />
+                <TezosAmount
+                  color="primary"
+                  size={ms(0.65)}
+                  amount={address.balance}
+                />
+              </ItemWrapper>
+            ))}
+          </CustomSelect>
+        </InvokeAddressContainer>
+        <ParametersContainer>
+          <TextField
+            label={t('components.interactModal.parameters')}
+            onChange={val => this.setState({ parameters: val })}
+          />
+        </ParametersContainer>
+        <RowContainer>
+          <ColContainer>
+            <TextField
+              type="number"
+              label={t('components.interactModal.storage_limit')}
+              onChange={val => this.setState({ storage: val })}
+            />
+          </ColContainer>
+          <ColContainer>
+            <TextField
+              type="number"
+              label={t('components.interactModal.gas_limit')}
+              onChange={val => this.setState({ gas: val })}
+            />
+          </ColContainer>
+        </RowContainer>
+        <RowContainer>
+          <AmountContainer>
+            <TezosNumericInput
+              decimalSeparator={t('general.decimal_separator')}
+              labelText={t('general.nouns.amount')}
+              amount={amount}
+              handleAmountChange={val => this.setState({ amount: val })}
+            />
+            <UseMax onClick={this.onUseMax}>
+              {t('general.verbs.use_max')}
+            </UseMax>
+          </AmountContainer>
+          <FeeContainer>
+            <Fees
+              low={averageFees.low}
+              medium={averageFees.medium}
+              high={averageFees.high}
+              fee={fee}
+              miniFee={OPERATIONFEE}
+              onChange={val => this.setState({ fee: val })}
+            />
+          </FeeContainer>
+        </RowContainer>
+
+        <PasswordButtonContainer>
+          {!isLedger && (
+            <PasswordInput
+              label={t('general.nouns.wallet_password')}
+              isShowed={isShowedPwd}
+              password={passPhrase}
+              changFunc={this.updatePassPhrase}
+              containerStyle={{ width: '60%', marginTop: '10px' }}
+              onShow={() => this.setState({ isShowedPwd: !isShowedPwd })}
+            />
+          )}
+          <InvokeButton
+            buttonTheme="primary"
+            disabled={isDisabled}
+            onClick={this.onInvokeOperation}
+          >
```

Comment:
I am unable to trigger this button by pressing the Enter key

Reply:
and also, can we clear up the inputs entered by the user after Invoke button is triggered and operation has successfully been sent?